views/index.pug

```diff
@@ -0,0 +1,18 @@
+html
+  head
+    title Kubercade Home
+    style
+      include style.css
+  body
+    #nav_bar
+      .navtab
+        a(href='#') Nav Link 1
+      .navtab
+        a(href='#') Nav Link 2
```

Comment:
You might get rid of the links here (just `a(href)`) so that the window location doesn't change to /# but you still get the button styling. I imagine we'll probably handle navigation by `onClick` handlers rather than `href`s.

Reply:
I was just going to keep them for now so the page doesn't break when the links are clicked, they can be removed when the `onClick` handlers are added.

Reply:
If you remove the href attribute completely, the a tag won't actually have link styling (underline, blue, etc.) and might not be clickable. Try setting it to `a(href)` which is what I used for the delete buttons on the large events posts. If that doesn't work, try `a(href='javascript:void(0);')` which similarly shouldn't change the url.

Reply:
`a(href)` implicitly resolves to `a(href="")`

Reply:
Changed to `href='javascript:void(0);'` in a78b164